src/Draco.Compiler/Api/Syntax/SyntaxTree.cs

```diff
@@ -157,7 +157,7 @@ public TNode FindInChildren<TNode>(int index = 0)
     /// Syntactically formats this <see cref="SyntaxTree"/>.
     /// </summary>
     /// <returns>The formatted tree.</returns>
-    public SyntaxTree Format() => Formatter.Format(this);
+    public string Format(FormatterSettings? settings = null) => Formatter.Format(this, settings);
```

Comment:
This is a significant change. I suppose the formatter doesn't build a tree anymore. For now this can stay, but I wonder if we want to expose a version returning string, or a tree.

Reply:
Due to the design of the formatter, I cannot return an AST, on the other hand I could return a token list if I refactor things.